README.md

```diff
@@ -23,7 +23,7 @@ Acceptance Tests As Monitors
 
 1. Include the atam4j maven dependency.
 
-2. Write Junit based tests in the usual manner, with the exception of including them in the main application classpath 
+2. Write Junit based tests in the usual manner, with the exception of adding a `@Monitor` annotation to the class and including them in the main application classpath 
```

Comment:
@TJ-Cutajar - Given the annotation isn't mandatory, we should update the sentence?